engine/admin/logging/logentries.md

```diff
@@ -0,0 +1,51 @@
+---
+aliases: ["/engine/reference/logging/logentries/"]
+title: "Logentries logging driver"
+description: "Describes how to use the logentries logging driver."
+keywords: ["logentries, docker, logging, driver"]
+---
+
+# Logentries logging driver
```

Comment:
FYI, the title for docs is now set in the metadata, under `title:` so you'll need to remove this.